string-manipulation-2/src/capitalize-words.js

```diff
@@ -0,0 +1,19 @@
+/* exported capitalizeWords */
+function capitalizeWords(string) {
+  var newUpperCasedString = '';
+  var arrayOfRawWords = string.split(' ');
+  for (var i = 0; i < arrayOfRawWords.length - 1; i++) {
+    newUpperCasedString += arrayOfRawWords[i][0].toUpperCase();
+    for (var j = 1; j < arrayOfRawWords[i].length; j++) {
+      newUpperCasedString += arrayOfRawWords[i][j].toLowerCase();
+    }
+    newUpperCasedString += ' ';
+  }
+  for (var h = arrayOfRawWords.length - 1; h < arrayOfRawWords.length; h++) {
+    newUpperCasedString += arrayOfRawWords[h][0].toUpperCase();
+    for (var k = 1; k < arrayOfRawWords[h].length; k++) {
+      newUpperCasedString += arrayOfRawWords[h][k].toLowerCase();
+    }
+  }
+  return newUpperCasedString;
+}
```

Comment:
We can simplify this a lot, so that way you don't need to do 4 `for` loops, because that will increase the runtime and overall complexity of this. Below is our solution on how we approached this
```
function capitalizeWords(string) {
  var words = string.split(' ');
  for (var i = 0; i < words.length; i++) {
    var word = words[i];
    var capitalized = word[0].toUpperCase() + word.slice(1).toLowerCase();
    words[i] = capitalized;
  }
  return words.join(' ');
}
```